SpatialGDK/Documentation/content/local-dev-workflow.md

```diff
@@ -0,0 +1,9 @@
+# Local Development Workflow
+
```

Comment:
It needs some sort of introduction and back-links to what local deployment means.